src/utils/helpers/structure/addReviewResponses.ts

```diff
@@ -0,0 +1,31 @@
+import { ReviewResponse } from '../../generated/graphql'
+import { FullStructure, SetReviewResponseOnElement } from '../../types'
+import getGroupedReviewResponses from './getGroupedReviewResponses'
+
+const addReviewResponses = (
```

Comment:
Maybe this should be renamed to `addPreviousAndLatestReviewResponses`, not super needed, just took me 5 minutes to try to figure out by the comment. It's not very obvious because of how this is passed to the function as props

Reply:
But it's also used for thisReviewResponses

in that case it would be `addPreviousOrLatestOrThisReviewResponses`, can rename to that if it's not too long